src/Input/Silk.NET.Input.Common/Structs/ScrollWheel.cs

```diff
@@ -3,14 +3,15 @@
 
 using System;
 using System.Globalization;
+using System.Runtime.CompilerServices;
 using System.Text;
 
 namespace Silk.NET.Input
 {
     /// <summary>
     /// Represents a scroll wheel.
     /// </summary>
-    public struct ScrollWheel
+    public readonly struct ScrollWheel : IEquatable<ScrollWheel>
```

Comment:
Making this readonly is a breaking change.

```suggestion
    public struct ScrollWheel : IEquatable<ScrollWheel>
```

Reply:
Oh and make all the methods readonly again!

Reply:
Thanks, I restored the previous state with 8fa05b4 - Prevent breaking change to Silk.NET.Input.ScrollWheel